docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -638,36 +638,35 @@
     "cuSpatial provides a simple sinusoidal longitude / latitude to Cartesian coordinate transform.  \n",
     "This function requires an origin point to determine the scaling parameters for the lonlat inputs.  \n",
     "\n",
-    "### [cuspatial.lonlat_to_cartesian](https://docs.rapids.ai/api/cuspatial/nightly/api_docs/spatial.html#cuspatial.lonlat_to_cartesian)\n",
+    "### [cuspatial.sinusoidal_projection](https://docs.rapids.ai/api/cuspatial/nightly/api_docs/spatial.html#cuspatial.sinusoidal_projection)\n",
     "\n",
     "The following cell converts the lonlat coordinates of the country of Afghanistan to Cartesian  \n",
     "coordinates in km, centered around the center of the country, suitable for graphing and display."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 1,
    "id": "a7a870dd-c0ae-41c1-a66c-cff4bd2db0ec",
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "            x           y\n",
-      "0  112.173577 -281.589872\n",
-      "1   62.152088 -280.852048\n",
-      "2   -5.573132 -257.390948\n",
-      "3  -33.070528 -243.848851\n",
-      "4  -98.002306 -279.540041\n"
+     "ename": "NameError",
```

Comment:
Shouldn't we have a way to automatically render the output of the API examples notebook in CI?